controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -2038,22 +2038,22 @@ int cli_bml::get_dcs_scan_results(const std::string &radio_mac, uint32_t max_res
                           << string_from_int_array(res.ap_EncryptionMode,
                                                    BML_CHANNEL_SCAN_ENUM_LIST_SIZE)
                           << std::endl
-                          << "  operating_frequency_band=" << res.ap_OperatingFrequencyBand
+                          << "  operating_frequency_band=" << int(res.ap_OperatingFrequencyBand)
                           << std::endl
                           << "  supported_standards="
                           << string_from_int_array(res.ap_SupportedStandards,
                                                    BML_CHANNEL_SCAN_ENUM_LIST_SIZE)
                           << std::endl
-                          << "  operating_standards=" << res.ap_OperatingStandards << std::endl
-                          << "  operating_channel_bandwidth=" << res.ap_OperatingChannelBandwidth
-                          << std::endl
-                          << "  beacon_period=" << (int)res.ap_BeaconPeriod << std::endl
-                          << "  noise=" << (int)res.ap_Noise << std::endl
-                          << "  basic_data_transfer_rates="
+                          << "  operating_standards=" << int(res.ap_OperatingStandards) << std::endl
+                          << "  operating_channel_bandwidth="
+                          << int(res.ap_OperatingChannelBandwidth) << std::endl
+                          << "  beacon_period_ms=" << (int)res.ap_BeaconPeriod << std::endl
```

Comment:
You could have converted the casts into initializers as well (nitpick).

Reply:
changed to functional cast instead